ts/webui/src/components/experiment/trialdetail/table/tableFunction/search/Search.tsx

```diff
@@ -142,102 +171,124 @@ function Search(props): any {
             alert('Please delete special characters in the conditions!');
             return;
         }
-        // according [input val] to change searchFilter list
-        const allFilterConditions = searchInputVal.trim().split(';');
         const newSearchFilter: any = [];
+        const str = searchInputVal.trim();
+        // user input: string and this string don't include [: < > ≠]
+        if (str.includes('>') || str.includes('<') || str.includes(':') || str.includes('≠')) {
+            // according [input val] to change searchFilter list
+            const allFilterConditions = searchInputVal.trim().split(';');
+            // delete '' in filter list
+            if (allFilterConditions.includes('')) {
```

Comment:
This only removes first empty string.
It should be `allFilterConditions.filter(item => item)`

Reply:
updated